jwql/utils/constants.py

```diff
@@ -215,6 +215,9 @@
 # Possible suffix types for guider exposures
 GUIDER_SUFFIX_TYPES = ['stream', 'stacked_uncal', 'image_uncal', 'stacked_cal', 'image_cal']
 
+# JWQL should ignore some filetypes in the filesystem.
+IGNORED_SUFFIXES = ['original', 'stream']
```

Comment:
Agreed with @bourque, have we always ignored `stream` files as well?

Reply:
I thought I saw one on an observation's page at one point. Anyway, I figure this can help us be sure to ignore them.